lib/rust/ensogl/component/grid-view/src/header.rs

```diff
@@ -365,6 +366,44 @@ impl<E: Entry, HeaderEntry: Entry<Params = E::Params>> GridView<E, HeaderEntry>
     }
 }
 
+impl<E: Entry, HeaderEntry: Entry<Params = E::Params>> FrpNetworkProvider
+    for GridView<E, HeaderEntry>
+{
+    fn network(&self) -> &frp::Network {
+        self.model.grid.network()
+    }
+}
```

Comment:
This is bad. I was writing multiple times in your PRs Mateusz that using structs with bounds does not make sense. Why do you need the bound `HeaderEntry: Entry<Params = E::Params>` in order to get the inner field? what is the purpose of having these bounds?